enterprise/backend/src/metabase_enterprise/embedding_hub/api.clj

```diff
@@ -10,9 +10,12 @@
    [toucan2.core :as t2]))
 
 (defn- has-user-added-database? []
-  (t2/exists? :model/Database {:where [:and
-                                       [:= :is_sample false]
-                                       [:= :is_audit false]]}))
+  (or (t2/exists? :model/Database {:where [:and
+                                           [:= :is_sample false]
+                                           [:= :is_audit false]]})
+      (t2/exists? :model/Table {:where [:and
+                                        [:= :active true]
+                                        [:= :is_upload true]]})))
```

Comment:
Is there a case where a Table exists without an accompanying Database? I remember working on some part that touches databases. And to have a table, you need a database.

This extra OR clause basically means it's a CSV upload table for a database where `is_sample = true` OR `is_audit = true` if I'm not mistaken. Can you verify this with some BE guys?

Reply:
@WiNloSt I got this snippet [from Roman](https://metaboat.slack.com/archives/C063Q3F1HPF/p1757927094096389). I believe the correct way to read this code is"is there a database that is not a sample database and not an audit (Instance Analytics) table, OR is there a table that is an uploads table (CSV) that is active?"

Reply:
Let me test this to confirm my theory